### PR TITLE
Reworded to make a bit easier to follow

### PR DESCRIPTION
**Pull request by cPlayIt**

### CODE REVIEW

File: doc/modules/language-guide/pages/basic-concepts.adoc [hunk 1]
```diff
@@ -199,8 +199,7 @@ Other languages that lack lexical scoping may give a different meaning to this p
 However, modern languages universally favor lexical scoping, the meaning given here.
 
 Aside from mathematical clarity, the chief practical benefit of lexical scoping is _security_, and its use in building compositionally-secure systems.
-Specifically, {proglang} gives very strong composition properties: Nesting your program within one you do not trust cannot, for example, arbitrarily re-associate your variable
-occurrences with different meanings.
+Specifically, {proglang} gives very strong composition properties: For example, nesting your program within one you do not trust cannot arbitrarily re-associate your variable occurrences with different meanings.
```
Comment (crusso) [inline doc/modules/language-guide/pages/basic-concepts.adoc:202]:
```suggestion
Specifically, {proglang} gives very strong composition properties: For example, nesting your program within one you do not trust cannot arbitrarily redefine your variables with different meanings.
```
Thanks, would this be even better?

**Review by crusso — APPROVED** [review]
Looks good to me. Thanks for the improvement!

File: doc/modules/language-guide/pages/basic-concepts.adoc (hunk 1 above)
Comment (crusso) [reply doc/modules/language-guide/pages/basic-concepts.adoc:202]: @matthewhammer whaddya think?

**Comment by cPlayIt** [comment]
Thanks, my first pull request on GitHub :)

I have a fix for the sdk web site as well, I know why the right hand TOC
isn't sticky as intended.

(Why it scrolls with rest of page erratically)

**Comment by crusso** [comment]
@cPlayIt I've just committed the suggestion and added the automerge-squash label for it to merge once CI succeeds. Thanks for th e contribution - I'm sure the sdk website team will be grateful too.

**Comment by cPlayIt** [comment]
Thanks!

**Comment by nomeata** [comment]
There were some merge conflicts. Too many PRs in parallel? I hope I resolved it correctly.

**Pull request closed without merging**